Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py

```diff
@@ -147,6 +149,36 @@ def git(cls, args, repository, additional_setup=None, hooks=None, **kwargs):
             sys.stderr.write('Failed to use {} as the merge strategy\n'.format('merge commits' if args.merge else 'rebase'))
             result += 1
 
+        if args.defaults:
+            pr_technique = repository.config(location='project')['webkitscmpy.pull-request']
+        else:
+            pr_technique = Terminal.choose(
+                'When editing pull-requests, would you like to overwrite existing commits or append to the branch?',
```

Comment:
Maybe it would be helpful to say something like:
```suggestion
                'When editing pull-requests, would you like to overwrite existing commits (like svn) or append to the branch (default behavior for git)?',
```
since I feel like what we're really asking is whether they want the semantics of `webkit-patch`.